.github/pull_request_template.md

```diff
@@ -20,8 +20,13 @@ Note that properly "linked issues" (either automatic links, or manual ones using
 ### Check list  
 * Is your feature branch up to date with the authoritative repository (NOAA/develop)?
 * Make sure you have checked the [checklist for a developer submitting to develop](https://github.com/NOAA-EMC/WW3/wiki/Code-Management#checklist-for-a-developer-submitting-to-develop) and [updating version number](https://github.com/NOAA-EMC/WW3/wiki/Code-Management#checklist-for-updating-version-number)
+* please add appropriate labels to indicate a change (The labels should be added by the code managemers):   
```

Comment:
Maybe: 
Please list appropriate labels code managers should add for this PR: _bug_, _documentation_, _enhancement_, _new feature_, ....